bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
     
 issues = bz.query(bz.url_to_query(query))
 
-print "[DEBUG] " + "Found " + str(len(issues)) + " bugzillas to process"
+if (options.verbose):
+    print "[DEBUG] " + "Found " + str(len(issues)) + " bugzillas to process"
```

Comment:
changed to info instead since its useful in default output to see how many issues was found.